packages/babel-types/src/definitions/index.ts

```diff
@@ -54,6 +55,7 @@ export {
   DEPRECATED_ALIASES,
   DEPRECATED_KEYS,
   NODE_PARENT_VALIDATIONS,
+  NODE_UNION_SHAPES__PRIVATE,
```

Comment:
```suggestion
  NODE_UNION_SHAPES__PRIVATE as "\u{10FFFF} PRIVATE!!! NODE_UNION_SHAPES",
```
:)

(if it's meant to be private, lets make it not a valid identifier, and `\u{10FFFF}` ensures autocompletion always shows it at the end)